src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -741,6 +792,24 @@ public class foo<T>
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InvokeWithOpenAngleCommitSeeOnSpace()
+        {
+            var markupBeforeCommit = @"class c
+{
+        /// <$$
+        void foo() { }
+}";
+
+            var expectedCodeAfterCommit = @"class c
+{
+        /// <see$$/>
+        void foo() { }
+}";
+
+            await VerifyCustomCommitProviderAsync(markupBeforeCommit, "see", expectedCodeAfterCommit, commitChar: ' ');
```

Comment:
That was the behaviour requested in #11489. Using `TAB` will still commit `<see cref="$$"/>` so hopefully it won't mess too much with peoples muscle memory.